gfx/map/map_object_data/generated/steppe_bush_01_generator.txt

```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
```

Comment:
> The tree density is incorrect for Quel'thalas, despite the map editor showing otherwise.

I don't know why it happens but it's for good.
The FPS drops when you look at Ashenvale.

I'll try to find why, but tree density in Ashenvale should be decreased.

Reply:
_I certainly was overzealous with tree placement. I'll reduce it to 40-70% depending on how it looks in the next commit._

Reply:
> _I certainly was overzealous with tree placement. I'll reduce it to 40-70% depending on how it looks in the next commit._

So, the reason may be `tree_leaf_01_mask` and `tree_leaf_02_single_mask`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132696503-e3a8ca1a-f828-4ede-a744-e8d86be0a6d6.png)
![image](https://user-images.githubusercontent.com/46384992/132696513-f3ab135a-1635-42c1-b8a5-d0f5bdb884ec.png)

</details>

Ashenvale uses the single mask and prob that's why it's so laggy. If it's true, I would stop using this mask at all.

Reply:
My FPS doesn't drop but keep in mind trees may affect the game a lot.